### PR TITLE
disco: add epoch forks

### PR DESCRIPTION
**Pull request by arjain4**

### CODE REVIEW

File: src/choreo/forks/fd_epoch_forks.h
Comment (lheeger-jump) [inline src/choreo/forks/fd_epoch_forks.h:1]: document this data structure! why is it needed, what is it for?

File: src/disco/store/fd_store.c [hunk 1]
```diff
@@ -226,7 +226,7 @@ fd_store_shred_insert( fd_store_t * store,
   if( FD_UNLIKELY( rc < FD_BLOCKSTORE_OK ) ) {
     FD_LOG_ERR( ( "failed to insert shred. reason: %d", rc ) );
   } else if ( rc == FD_BLOCKSTORE_OK_SLOT_COMPLETE ) {
-    FD_LOG_WARNING(("BLOCK COMPLETE"));
+    FD_LOG_DEBUG(("BLOCK COMPLETE"));
```
Comment (lheeger-jump) [inline src/disco/store/fd_store.c:229]: rm this log

**Review by lheeger-jump — CHANGES_REQUESTED** [review]
Please add a stub unit test like the new `test_trusted_slots.c` and add it to Local.mk

**Review by lidatong — CHANGES_REQUESTED** [review]
can you move out of choreo into disco? choreo is really just meant for consensus rules

otherwise looks good superficially to me

**Review by lheeger-jump — APPROVED** [review]

**Pull request merged**